reviewrot/gerritstack.py

```diff
@@ -52,15 +51,15 @@ def request_reviews(self, host, repo_name, age=None,
         # checks if specified repo exists
         repo_exists = self.check_repo_exists(repo_name, ssl_verify)
 
-        if self.host_exists and repo_exists:
-            request_url = "{}/changes/?q=project:{}+status:open&" \
-                          "o=DETAILED_ACCOUNTS".format(self.url, repo_name)
-            log.debug('Looking for change requests for %s -> %s',
-                      self.url, repo_name)
-            review_response = self._call_api(url=request_url,
-                                             ssl_verify=ssl_verify)
-            reviews = self.format_response(review_response, age, show_last_comment)
-        return reviews
+        if not (self.host_exists and repo_exists):
```

Comment:
I would rephrase this as `if not self.host_exists or not repo_exists`, just so readers don't have to perform De Morgan's in their heads :)

Reply:
Done.